ticket_facts.view.lkml

```diff
@@ -0,0 +1,277 @@
+           WHEN ${via__channel} = 'mobile_sdk' AND ${via__source__rel} = 'mobile_sdk' THEN 'Inbound Email (from GA App)'
+           WHEN ${via__channel} = 'facebook' AND ${via__source__rel} IN ('message', 'post') THEN 'Facebook Message'
+           WHEN ${via__channel} = 'twitter' AND ${via__source__rel} IN ('direct_message', 'mention') THEN 'Twitter Message'
+           WHEN ${via__channel} = 'web' AND ${via__source__rel} = 'follow_up' THEN 'Follow-up Ticket'
+           WHEN ${via__channel} = 'web' AND ${via__source__rel} IS NULL AND ${number_outbound_emails} > 0 THEN 'Outbound Email'
+           ELSE NULL END;;
+    }
+
+    dimension: group_name {
+      description: "Ticket's group name. (If ticket was in more than one group, the group name defaults to the last assigned group)"
+      type: string
+      sql: ${TABLE}.name ;;
+    }
+
+    dimension: via__source__to__name {
+      hidden: yes
+      type: string
+      sql: ${TABLE}.via__source__to__name ;;
+    }
+
+    dimension: multiple_groups_involved {
+      description: "Yes, if the ticket has changed assigned group at any point."
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${TABLE}.number_of_groups_involved > 0 ;;
+    }
+
+    dimension: number_outbound_calls {
+      description: "Number of outbound calls associated with ticket"
+      group_label: "Activity Counts"
+      type: number
+      sql: ${TABLE}.number_outbound_calls ;;
+    }
+
+    dimension: has_outbound_calls {
+      description: "Yes, if ticket has associated outbound calls"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_outbound_calls} > 0 ;;
+    }
+
+    dimension: number_inbound_calls {
+      description: "Number of inbound calls associated with ticket"
+      group_label: "Activity Counts"
+      type: number
+      sql: ${TABLE}.number_inbound_calls ;;
+    }
+
+    dimension: has_inbound_calls {
+      description: "Yes, if ticket has associated inbound calls"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_inbound_calls} > 0 ;;
+    }
+
+    dimension: number_inbound_emails {
+      description: "Number of inbound emails associated with ticket"
+      group_label: "Activity Counts"
+      type: number
+      sql: ${TABLE}.number_inbound_emails ;;
+    }
+
+    dimension: has_inbound_emails {
+      description: "Yes, if ticket has associated inbound emails"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_inbound_emails} > 0 ;;
+    }
+
+    dimension: involves_customer {
+      description: "Yes, if ticket has inbound or outbound activity"
+      group_label: "Ticket Details"
+      type: yesno
+      sql: ${number_inbound_calls} > 0 OR ${number_inbound_emails} > 0 OR ${number_outbound_calls} > 0 OR ${number_outbound_emails} > 0 ;;
+    }
+
+    dimension: is_first_touch_resolution_email {
```

Comment:
In Zendesk, they call this "One Touch Resolution":

https://support.zendesk.com/hc/en-us/community/posts/206723627-One-Touch-Resolution-Calculation

So perhaps `is_one_touch_resolved`?

Reply:
done